instance/models/mixins/openedx_config.py

```diff
@@ -153,6 +153,7 @@ def _get_configuration_variables(self):
 
             # Nginx
             "NGINX_SET_X_FORWARDED_HEADERS": False,
+            "NGINX_PROXY_ORIGIN_REGEX": r'^(http|https)://.*\.{}'.format(self.instance.domain),
```

Comment:
As per my understanding `self.instance.domain` will be set to base domain for eg. `navin.opencraft.hosting`.
So the regex will match any url starting with http/https and ending with the `navin.opencraft.hosting`.